diff --git a/node_modules/@herefishyfish/nativescript-sqlite-rxstorage-adapter/index.js b/node_modules/@herefishyfish/nativescript-sqlite-rxstorage-adapter/index.js
index adf40c2..0bfcc22 100644
--- a/node_modules/@herefishyfish/nativescript-sqlite-rxstorage-adapter/index.js
+++ b/node_modules/@herefishyfish/nativescript-sqlite-rxstorage-adapter/index.js
@@ -1,5 +1,7 @@
+import $debug from '@/debug.yml';
 import { knownFolders, path } from '@nativescript/core';
 import { openOrCreate } from '@herefishyfish/requery-sqlite';
+
 export const getSQLiteBasicsNativeScript = (options) => {
     return {
         setPragma: async (db, key, value) => {
@@ -12,9 +14,11 @@ export const getSQLiteBasicsNativeScript = (options) => {
             });
         },
         all: async (db, queryWithParams) => {
+            if($debug.rxdb) console.warn('adapter.all',queryWithParams)
             return (await db.select(queryWithParams.query, queryWithParams.params ?? []));
         },
         run: async (db, queryWithParams) => {
+            if($debug.rxdb) console.warn('adapter.run',queryWithParams)
             await db.select(queryWithParams.query, queryWithParams.params ?? []);
         },
         close: (db) => {
